dvc/commands/plots.py

```diff
@@ -222,14 +222,9 @@ def run(self):
 
 
 class CmdPlotsTemplates(CmdBase):
-    TEMPLATES_CHOICES = [
-        "simple",
-        "linear",
-        "confusion",
-        "confusion_normalized",
-        "scatter",
-        "smooth",
-    ]
+    from dvc_render.vega_templates import TEMPLATES
+
+    TEMPLATES_CHOICES = [t.DEFAULT_NAME for t in TEMPLATES]
```

Comment:
Please move this to file level, there is no difference except this pollutes the class namespace.